lib/ecto/adapters/myxql/connection.ex

```diff
@@ -1080,6 +1091,9 @@ if Code.ensure_loaded?(MyXQL) do
       [?`, name, ?`]
     end
 
+    defp format_to_sql(:json), do: "FORMAT=JSON"
+    defp format_to_sql(:traditional), do: "FORMAT=TRADITIONAL"
```

Comment:
```suggestion
    defp format_to_sql(:text), do: "FORMAT=TRADITIONAL"
```

We should use the same option names across adapters if possible. Traditional can be thought of as the text format, I'm pretty sure?

Reply:
Yes, true. Fixing.